src/useEnigma.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+export default function useEnigma(doc, objectId) {
```

Comment:
I don't think useEnigma is the best name for this custom hook, as it returns a specific qlikObjects data/ chart data.
This makes me thing I'll get back an instance of the top level Enigma class, which handles to the qlik server/ connects to specific qlik apps.